include/armadillo_bits/auxlib_meat.hpp

```diff
@@ -1883,6 +1883,56 @@ auxlib::chol_band_common(Mat<eT>& X, const uword KD, const uword layout)
   }
 
 
+//
+// hessenberg decomposition
+template<typename eT, typename T1>
+inline
+bool
+auxlib::hess(Mat<eT>& H, const Base<eT,T1>& X, Col<eT>& tao)
+  {
+  arma_extra_debug_sigprint();
+  
+  #if defined(ARMA_USE_LAPACK)
+    {
+    H = X.get_ref();
+    
+    arma_debug_check( (H.is_square() == false), "hess(): given matrix must be square sized" );
+    
+    if(H.is_empty())
+      {
+      H.reset();
+      return true;
+      }
+    
+    arma_debug_assert_blas_size(H);
+    
+    const uword H_n_rows = H.n_rows;
+    
+    blas_int  n      = H_n_rows;
```

Comment:
`H_n_rows` must be cast to a `blas_int` before assignment to `n`.
For example: `blas_int n = blas_int(H_n_rows);`